python/nnvm/testing/darknet.py

```diff
@@ -0,0 +1,734 @@
+# pylint: disable=invalid-name, unused-variable, unused-argument, old-style-class, no-init
+"""
+Compile DarkNet Models
+====================
+DarkNet helper functions for yolo and image loading.
```

Comment:
add comment that this cannot be loaded by default and is only used in test cases